code/script.js

```diff
@@ -2,6 +2,10 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutButton = document.getElementById('filter')
+const winOrLose = document.getElementById(`winOrLose`)
+const winOrLoseText = document.getElementById(`winOrLoseText`)
+const playAgainButton = document.getElementById(`playAgain`)
```

Comment:
As it is not a template string you can use single quotation marks instead :) Just a styling thing :)